src/main/java/com/microsoft/azure/functions/worker/broker/OverloadResolver.java

```diff
@@ -69,22 +53,21 @@ else if (overloadComparator.compare(possibleInvokes.get(0), possibleInvokes.get(
                     argument = dataStore.getDataByType(param.type);
                 }
                 BindingData actualArg = argument.orElseThrow(WrongMethodTypeException::new);
-                invokeInfo.matchingLevelCount[actualArg.getLevel().getIndex()]++;
                 invokeInfo.appendArgument(actualArg.getValue());
             }
             if (!method.entry.getReturnType().equals(void.class) && !method.entry.getReturnType().equals(Void.class)) {
                 dataStore.getOrAddDataTarget(invokeInfo.outputsId, BindingDataStore.RETURN_NAME, method.entry.getReturnType());
             }
-            return Optional.of(invokeInfo);
+            return invokeInfo;
         } catch (Exception ex) {
-            return Optional.empty();
+            //TODO log
+            return null;
```

Comment:
Planning on addressing this todo in thisi PR?

Reply:
Still figuring out  which exceptions are useful to log. Will address this separately.